superset-frontend/src/components/ImportModal/index.tsx

```diff
@@ -16,10 +16,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Upload } from 'antd';
```

Comment:
A few of us were just talking about antd imports the other day. Do you mind importing and exporting this component in the /superset-frontend/src/common/components/index.tsx file and pulling it in from there? We're going to try to clean all these up at some point for consistency.

Reply:
Yeah, I like having that abstraction layer for the day we move from `antd` to the new fancy component library. Will move.